openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js

```diff
@@ -335,7 +335,7 @@ export class OpenC3Api {
       }
     } else {
       data = await this.exec('tlm', [target_name, packet_name, item_name], {
-        type: data_type,
+        type: value_type,
```

Comment:
We're not at all consistent with the term `data_type` vs `value_type` throughout the code. I assume you're indicating you like value_type going forward.